hack/.golint_failures

```diff
@@ -422,23 +422,9 @@ pkg/volume/util/fs
 pkg/volume/util/recyclerclient
 pkg/volume/util/volumepathhandler
 pkg/volume/vsphere_volume
-plugin/pkg/admission/antiaffinity
-plugin/pkg/admission/eventratelimit/apis/eventratelimit
 plugin/pkg/admission/eventratelimit/apis/eventratelimit/v1alpha1
```

Comment:
can we remove this as well. this comment also applies for L437-438

Reply:
Golint still shows failures for these in my local machine. There are non-autogenerated files called `defaults.go` with golint violations because they invoke autogenerated function names. Example:
```
plugin/pkg/admission/eventratelimit/apis/eventratelimit/v1alpha1/defaults.go:25:6: don't use underscores in Go names; func SetDefaults_Configuration should be SetDefaultsConfiguration
```
I can try and see if the CI will ignore these violations.

Reply:
> There are non-autogenerated files called defaults.go with golint violations because they invoke autogenerated function names.

oh. apologises. lets keep it if it's generated code

Reply:
Some of the complaints can be fixed, but the ones on using underscores require changing how code is generated. Failure results:
```
Errors from golint:
plugin/pkg/admission/eventratelimit/apis/eventratelimit/v1alpha1/defaults.go:25:1: exported function SetDefaults_Configuration should have comment or be unexported
plugin/pkg/admission/eventratelimit/apis/eventratelimit/v1alpha1/defaults.go:25:6: don't use underscores in Go names; func SetDefaults_Configuration should be SetDefaultsConfiguration
plugin/pkg/admission/resourcequota/apis/resourcequota/v1alpha1/defaults.go:25:1: exported function SetDefaults_Configuration should have comment or be unexported
plugin/pkg/admission/resourcequota/apis/resourcequota/v1alpha1/defaults.go:25:6: don't use underscores in Go names; func SetDefaults_Configuration should be SetDefaultsConfiguration
plugin/pkg/admission/resourcequota/apis/resourcequota/v1beta1/defaults.go:25:1: exported function SetDefaults_Configuration should have comment or be unexported
plugin/pkg/admission/resourcequota/apis/resourcequota/v1beta1/defaults.go:25:6: don't use underscores in Go names; func SetDefaults_Configuration should be SetDefaultsConfiguration
```

Reply:
I removed the commit that verified this breaks in CI.